README.md

```diff
@@ -91,9 +91,9 @@ If a development version is not to be used (normal case), you can drop the optio
 >
 > This command will install the following packages:
 >
-> * `cmsgemos` version 0.3.1 (`-c 0.3.1`)
-> * `gemplotting` version 1.0.0-dev5 (`-g 1.0.0 -G 5`)
-> * `vfatqc` version 2.0.0-dev3 (`-v 2.0.0 -V 3`)
+> * [cmsgemos](https://github.com/cms-gem-daq-project/cmsgemos/tags) version 0.3.1 (`-c 0.3.1`)
+> * [gemplotting](https://github.com/cms-gem-daq-project/gem-plotting-tools/tags) version 1.0.0-dev5 (`-g 1.0.0 -G 5`)
+> * [vfatqc](https://github.com/cms-gem-daq-project/vfatqc-python-scripts/tags) version 2.0.0-dev3 (`-v 2.0.0 -V 3`)
```

Comment:
Adding the links as you’ve done so here don’t really make sense since you’re providing the general link but a specific tag. Please see my original request for clarification.